src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf

```diff
@@ -8,51 +8,51 @@
             </trans-unit>
             <trans-unit id="2">
                 <source>Authentication credentials could not be found.</source>
-                <target>Authentication credentials could not be found.</target>
+                <target>פרטי זיהוי לא נמצאו.</target>
             </trans-unit>
             <trans-unit id="3">
                 <source>Authentication request could not be processed due to a system problem.</source>
                 <target>לא ניתן היה לעבד את בקשת אימות בגלל בעיית מערכת.</target>
             </trans-unit>
             <trans-unit id="4">
                 <source>Invalid credentials.</source>
-                <target>שם משתמש או סיסמא שגויים</target>
+                <target>שם משתמש או סיסמא שגויים.</target>
             </trans-unit>
             <trans-unit id="5">
                 <source>Cookie has already been used by someone else.</source>
-                <target>Cookie has already been used by someone else.</target>
+                <target>עוגיה כבר שומשה.</target>
             </trans-unit>
             <trans-unit id="6">
                 <source>Not privileged to request the resource.</source>
-                <target>Not privileged to request the resource.</target>
+                <target>אין הרשאה מתאימה.</target>
             </trans-unit>
             <trans-unit id="7">
                 <source>Invalid CSRF token.</source>
-                <target>Invalid CSRF token.</target>
+                <target>אסימון CSRF לא חוקי.</target>
             </trans-unit>
             <trans-unit id="9">
                 <source>No authentication provider found to support the authentication token.</source>
-                <target>No authentication provider found to support the authentication token.</target>
+                <target>לא נמצא ספק אימות המתאימה לבקשה.</target>
             </trans-unit>
             <trans-unit id="10">
                 <source>No session available, it either timed out or cookies are not enabled.</source>
                 <target>אין סיישן זמין, או שתם הזמן הקצוב או העוגיות אינן מופעלות.</target>
             </trans-unit>
             <trans-unit id="11">
                 <source>No token could be found.</source>
-                <target>No token could be found.</target>
+                <target>הטוקן לא נמצא.</target>
             </trans-unit>
             <trans-unit id="12">
                 <source>Username could not be found.</source>
-                <target>שם משתמש לא נמצא במערכת</target>
+                <target>שם משתמש לא נמצא במערכת.</target>
```

Comment:
FYI, We dont need to write "on system". A simple "Username not found" is good enough. 

Do whatever sounds better.

Reply:
You can remove the word
במערכת

Reply:
Yeah, I think that is better. 

Thank you